docs/topics/proc_deploying-a-springboot-application-using-war-files.adoc

```diff
@@ -30,13 +30,34 @@ You must configure your build and deployment settings to ensure that your applic
   ...
   <packaging>war</packaging>
   ...
+</project>
 ----
 
-. Ensure the the `repackage` Maven goal for the Spring Boot Maven plugin is defined in the `pom.xml` file.
+. Specify `spring-boot-starter-tomcat` as a dependency of your application:
 +
 .Example `pom.xml`
 [source,xml,options="nowrap",subs="attributes+"]
 ----
+<project ...>
+  ...
+  <dependencies>
+    ...
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-tomcat</artifactId>
```

Comment:
@liborfuka Would adding `spring-boot-starter-tomcat` as a dependency be  the correct way of specifying an application server?
Or would it be better to specify the productized JWS Tomcat artifact directly?

Reply:
Its correct way to add tomcat as dependecny.

Reply:
@liborfuka thank you